diff --git a/node_modules/aframe/src/utils/device.js b/node_modules/aframe/src/utils/device.js
index 1e6930c..04ac190 100644
--- a/node_modules/aframe/src/utils/device.js
+++ b/node_modules/aframe/src/utils/device.js
@@ -33,7 +33,7 @@ if (isWebXRAvailable) {
     navigator.xr.isSessionSupported('immersive-ar').then(function (supported) {
       supportsARSession = supported;
       updateEnterInterfaces();
-    }).catch(function () {});
+    }).catch(function () { });
   } else if (navigator.xr.supportsSession) {
     // Fallback for implementations that haven't updated to the new spec yet,
     // the old version used supportsSession which is rejected for missing
@@ -45,7 +45,7 @@ if (isWebXRAvailable) {
     navigator.xr.supportsSession('immersive-ar').then(function () {
       supportsARSession = true;
       updateEnterInterfaces();
-    }).catch(function () {});
+    }).catch(function () { });
   } else {
     error('WebXR has neither isSessionSupported or supportsSession?!');
   }
@@ -54,13 +54,13 @@ if (isWebXRAvailable) {
 /**
  * Determine if a headset is connected.
  */
-export function checkHeadsetConnected () {
+export function checkHeadsetConnected() {
   return supportsVRSession || supportsARSession;
 }
 
-export function checkARSupport () { return supportsARSession; }
+export function checkARSupport() { return supportsARSession; }
 
-export function checkVRSupport () { return supportsVRSession; }
+export function checkVRSupport() { return supportsVRSession; }
 
 /**
  * Checks if browser is mobile and not stand-alone dedicated vr device.
@@ -88,7 +88,7 @@ export var isMobile = (function () {
  *  Detect tablet devices.
  *  @param {string} mockUserAgent - Allow passing a mock user agent for testing.
  */
-export function isTablet (mockUserAgent) {
+export function isTablet(mockUserAgent) {
   var userAgent = mockUserAgent || window.navigator.userAgent;
 
   var isTablet = /Nexus (7|9)|xoom|sch-i800|playbook|tablet|kindle/i.test(userAgent);
@@ -103,7 +103,7 @@ export function isTablet (mockUserAgent) {
  *  @param {string} mockDevicePlatform - Allow passing a mock device platform for testing.
  *  @param {string} mockDeviceTouchPoints - Allow passing a mock device touch points for testing.
 */
-export function isIpad (mockUserAgent, mockDevicePlatform, mockDeviceTouchPoints) {
+export function isIpad(mockUserAgent, mockDevicePlatform, mockDeviceTouchPoints) {
   var userAgent = mockUserAgent || window.navigator.userAgent;
   var platform = mockDevicePlatform || window.navigator.platform;
   var maxTouchPoints = mockDeviceTouchPoints || window.navigator.maxTouchPoints || 0;
@@ -114,7 +114,7 @@ export function isIpad (mockUserAgent, mockDevicePlatform, mockDeviceTouchPoints
 /**
  *  Detect Apple Vision Pro devices.
  */
-export function isAppleVisionPro () {
+export function isAppleVisionPro() {
   // Safari for Apple Vision Pro presents itself as a desktop browser.
   var isMacintosh = navigator.userAgent.includes('Macintosh');
   // Discriminates between a "real" desktop browser and Safari for Vision Pro.
@@ -124,36 +124,36 @@ export function isAppleVisionPro () {
   return isMacintosh && hasFiveTouchPoints && isWebXRAvailable;
 }
 
-export function isIOS () {
+export function isIOS() {
   return /iPad|iPhone|iPod/.test(window.navigator.platform);
 }
 
-export function isMobileDeviceRequestingDesktopSite () {
+export function isMobileDeviceRequestingDesktopSite() {
   return !isMobile() && !isMobileVR() && window.orientation !== undefined;
 }
 
 /**
  *  Detect Oculus Browser (standalone headset)
  */
-export function isOculusBrowser () {
+export function isOculusBrowser() {
   return /(OculusBrowser)/i.test(window.navigator.userAgent);
 }
 
 /**
  *  Detect Firefox Reality (standalone headset)
  */
-export function isFirefoxReality () {
+export function isFirefoxReality() {
   return /(Mobile VR)/i.test(window.navigator.userAgent);
 }
 
 /**
  *  Detect browsers in Stand-Alone headsets
  */
-export function isMobileVR () {
+export function isMobileVR() {
   return isOculusBrowser() || isFirefoxReality() || isAppleVisionPro();
 }
 
-export function isR7 () {
+export function isR7() {
   return /R7 Build/.test(window.navigator.userAgent);
 }
 
@@ -161,7 +161,7 @@ export function isR7 () {
  * Checks mobile device orientation.
  * @returns {boolean} True if landscape orientation.
  */
-export function isLandscape () {
+export function isLandscape() {
   var orientation = window.orientation;
   if (isR7()) { orientation += 90; }
   return orientation === 90 || orientation === -90;
@@ -171,8 +171,9 @@ export function isLandscape () {
  * Check if running in a browser or spoofed browser (bundler).
  * We need to check a node api that isn't mocked on either side.
  * `require` and `module.exports` are mocked in browser by bundlers.
+ * process is polyfilled with process.browser true by Next.js with "use client".
  */
-export var isBrowserEnvironment = typeof process === 'undefined';
+export var isBrowserEnvironment = typeof process === 'undefined' || process.browser === true;
 
 /**
  * Check if running in node on the server.
